app/configurations/config.matka.js

```diff
@@ -510,6 +511,11 @@ export default {
       sv: 'Raseborg',
       en: 'Raasepori',
     },
+    VARELY: {
+      fi: 'Seutu+',
+      sv: 'Seutu+',
+      en: 'Seutu+',
+    },
```

Comment:
Not quite sure what to call this but probably Varsinais-Suomi. Seutu+ is more than this data set (and it sounds slightly weird if you are not aware of what it is and you are interested in alerts around Rovaniemi region for example :D).